src/ws/constants.ts

```diff
@@ -3,7 +3,7 @@ export const KINDS = {
   FEEDBACK: 'feedback',
   ITEM: 'item',
   OWN: 'own',
-  RECYCLEBIN: 'recyclebin',
+  RECYCLE_BIN: 'recycle_bin',
```

Comment:
No the string constant cannot be changed, it's the API with the back-end.

Reply:
@dialexo okay, but I can change the label only ?